keyboards/db/db63/info.json

```
@@ -0,0 +1,18 @@
+{
+    "keyboard_name": "db63",
+    "url": "http://www.keyboard-layout-editor.com/#/gists/dadea703fc8bfc87dc7c480de9f3ef38",
+    "maintainer": "QMK Community",
+    "width": 15,
+    "height": 5,
+    "layouts": {
+        "LAYOUT": {
+            "layout": [
+                { "label": "~", "x": 0, "y": 0 }, { "label": "!", "x": 1, "y": 0 }, { "label": "@", "x": 2, "y": 0 }, { "label": "#", "x": 3, "y": 0 }, { "label": "$", "x": 4, "y": 0 }, { "label": "%", "x": 5, "y": 0 }, { "label": "^", "x": 6, "y": 0 }, { "label": "&", "x": 7, "y": 0 }, { "label": "*", "x": 8, "y": 0 }, { "label": "(", "x": 9, "y": 0 }, { "label": ")", "x": 10, "y": 0 }, { "label": "_", "x": 11, "y": 0 }, { "label": "+", "x": 12, "y": 0 }, { "label": "Backspace", "x": 13, "y": 0, "w": 2 },
+                { "label": "Tab", "x": 0, "y": 1, "w": 1.5 }, { "label": "Q", "x": 1.5, "y": 1 }, { "label": "W", "x": 2.5, "y": 1 }, { "label": "E", "x": 3.5, "y": 1 }, { "label": "R", "x": 4.5, "y": 1 }, { "label": "T", "x": 5.5, "y": 1 }, { "label": "Y", "x": 6.5, "y": 1 }, { "label": "U", "x": 7.5, "y": 1 }, { "label": "I", "x": 8.5, "y": 1 }, { "label": "O", "x": 9.5, "y": 1 }, { "label": "P", "x": 10.5, "y": 1 }, { "label": "{", "x": 11.5, "y": 1 }, { "label": "}", "x": 12.5, "y": 1 }, { "label": "|", "x": 13.5, "y": 1, "w": 1.5 },
+                { "label": "Fn3", "x": 0, "y": 2, "w": 1.75 }, { "label": "A", "x": 1.75, "y": 2 }, { "label": "S", "x": 2.75, "y": 2 }, { "label": "D", "x": 3.75, "y": 2 }, { "label": "F", "x": 4.75, "y": 2 }, { "label": "G", "x": 5.75, "y": 2 }, { "label": "H", "x": 6.75, "y": 2 }, { "label": "J", "x": 7.75, "y": 2 }, { "label": "K", "x": 8.75, "y": 2 }, { "label": "L", "x": 9.75, "y": 2 }, { "label": ":", "x": 10.75, "y": 2 }, { "label": "\"", "x": 11.75, "y": 2 }, { "label": "Enter", "x": 12.75, "y": 2, "w": 2.25 },
+                { "label": "Shift", "x": 0, "y": 3, "w": 2.25 }, { "label": "Z", "x": 2.25, "y": 3 }, { "label": "X", "x": 3.25, "y": 3 }, { "label": "C", "x": 4.25, "y": 3 }, { "label": "V", "x": 5.25, "y": 3 }, { "label": "B", "x": 6.25, "y": 3 }, { "label": "N", "x": 7.25, "y": 3 }, { "label": "M", "x": 8.25, "y": 3 }, { "label": "<", "x": 9.25, "y": 3 }, { "label": ">", "x": 10.25, "y": 3 }, { "label": "?", "x": 11.25, "y": 3 }, { "label": "Shift", "x": 12.25, "y": 3, "w": 1.75 },
```

[Your header file](https://github.com/qmk/qmk_firmware/blob/7b466e88f51ae705edcfe135e04bd3269d93ef03/keyboards/db/db63/db63.h#L37-L41) specifies 63 keys, but you have 61 keys defined in `info.json`, which means your QMK Configurator implementation is incorrect.

```suggestion
                { "label": "Shift", "x": 0, "y": 3, "w": 2.25 }, { "label": "Z", "x": 2.25, "y": 3 }, { "label": "X", "x": 3.25, "y": 3 }, { "label": "C", "x": 4.25, "y": 3 }, { "label": "V", "x": 5.25, "y": 3 }, { "label": "B", "x": 6.25, "y": 3 }, { "label": "N", "x": 7.25, "y": 3 }, { "label": "M", "x": 8.25, "y": 3 }, { "label": "<", "x": 9.25, "y": 3 }, { "label": ">", "x": 10.25, "y": 3 }, { "label": "Shift", "x": 11.25, "y": 3, "w": 1.75 }, { "label": "Up", "x": 13, "y": 3 }, { "label": "Fn2", "x": 14, "y": 3 },
```

---

thanks, i updated it